message_definitions/v1.0/ASLUAV.xml

```diff
@@ -270,5 +270,24 @@
       <field type="uint8_t" name="sinr_ecio">SINR (LTE) or ECIO (WCDMA) as reported by modem (unconverted)</field>
       <field type="uint8_t" name="rsrq">RSRQ (LTE only) as reported by modem (unconverted)</field>
     </message>
+    <message id="214" name="SATCOM_LINK_STATUS">
+      <description>Status of the SatCom link</description>
+      <field type="uint64_t" name="timestamp" units="us">Timestamp</field>
+      <field type="uint64_t" name="last_heartbeat" units="us">Timestamp of the last successful sbd session</field>
+      <field type="uint16_t" name="failed_sessions">Number of failed sessions</field>
+      <field type="uint16_t" name="successful_sessions">Number of successful sessions</field>
+      <field type="uint8_t" name="signal_quality">Signal quality</field>
+      <field type="uint8_t" name="ring_pending">Ring call pending</field>
+      <field type="uint8_t" name="tx_session_pending">Transmission session pending</field>
+      <field type="uint8_t" name="rx_session_pending">Receiving session pending</field>
+    </message>
+    <message id="215" name="SENSOR_AIRFLOW_ANGLES">
```

Comment:
This looks like a different form of ArduPilot's `AOA_SSA` message (@samuelctabor).  Except a couple of bytes of waste (float means you can use NaN for invalid....)

I mean this is in a dialect and all, so it doesn't affect ArduPilot, but it does show mavlink has a ways to go as a standard.